lib/slack_it.rb

```diff
@@ -1,37 +1,34 @@
 # require "slack_it/version"
+require "invoke_call/version"
```

Comment:
yeah we should just require "slack_it/version", invoke_call is a different project